src/interpreter.test.ts

```diff
@@ -2925,6 +2926,10 @@ test('sheetRequiresAdjudication triggers only if both sides are blank ballot', a
     },
   }
 
+  const sideTrulyBlank: BlankPage = {
+    type: 'BlankPage',
+  }
```

Comment:
Is this from #157?

Reply:
yes, sorry, I merged that PR into here cause they were chained.